### PR TITLE
Partner blackList

### PR DESCRIPTION
**Pull request by nicolasdoclot**
This module allows you to add a category 'blacklist' in the form of your customers. If a customer is in this category it will appear in red in OpenERP.

### CODE REVIEW

File: partner_blacklist/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+{
+	'name': 'blacklist partner',
+	'version': '1.0',
+	'category': 'Generic Modules/Others',
+	'description': """
+	This module allows you to add a category 'blacklist' in the form of your customers.
+	If a customer is in this category it will appear in red in OpenERP.
+	""",
```
Comment (gurneyalex) [inline partner_blacklist/__openerp__.py:28]: description can be removed in 8.0

File: partner_blacklist/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	'author': 'BHC',
```
Comment (gurneyalex) [inline partner_blacklist/__openerp__.py:29]: please add Odoo Community Association (OCA) to authors

**Comment by gurneyalex** [comment]
@nicolasdoclot hello. AFAICT, BHC does not have an enterprise CLA with the OCA. Can you send an individual CLA? 

Additionally there are a few coding conventions not met by this addon and found by flake8 (see https://travis-ci.org/OCA/partner-contact/jobs/76109300#L172). Could you fix these?

**Comment by rafaelbn** [comment]
Hi @nicolasdoclot do you want to continue with this PR? You have to sign CLA. If not please confirm us and I will close this PR.

**Comment by rafaelbn** [comment]
No answer

**Pull request closed without merging**